src/server/migrations/0.7.0-0.7.1/sql/readings/create_materialized_hourly_readings.sql

```diff
@@ -0,0 +1,36 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+DROP VIEW IF EXISTS hourly_readings;
```

Comment:
I thought 'CREATE OR REPLACE' would not work because we would be replacing an unmaterialized view with a  materialized view? I did this in two steps because I thought I had heard this somewhere, perhaps in one of our meetings. I think this is fine though, but I can change it if you think it'll work in one step.

Reply:
Yes, this seems reasonable. It is unlikely someone would run this twice (not sure if our migrations can all do that) so it just means recreating the data which won't take that long.